com.unity.shadergraph/Editor/Drawing/PreviewManager.cs

```diff
@@ -716,13 +716,22 @@ void RenderPreview(PreviewRenderData renderData, Mesh mesh, Matrix4x4 transform)
             }
         }
 
+
+        private int srpInitAttempts = 0;
         void InitializeSRPIfNeeded()
         {
             if ((Shader.globalRenderPipeline != null) && (Shader.globalRenderPipeline.Length > 0))
             {
                 return;
```

Comment:
Should we reset srpInitAttempts here just in case the comment below ever becomes untrue?